server/pulp/server/util.py

```diff
@@ -182,3 +185,66 @@ def __init__(self, obj, filter=()):
         for k,v in obj.items():
             if k in filter:
                 self[k] = v
+
+def copytree(src, dst, symlinks=False, ignore=None):
+    """
+    Copies src tree to dst
+
+    shutil.copytree method uses copy2() and copystat() to perform the recursive copy of a
+    directory. Both of these methods copy the attributes associated with the files. We are When copying
+    files from /var/cache/pulp to /var/lib/pulp, we don't want to copy the SELinux security context
+    labels. See: https://pulp.plan.io/issues/1253
```

Comment:
I feel the doc-block explains the reason for the method's existence well enough to not need a reference to the issue. That URL may one day not be around and leave the reader wondering what else they might be missing. Personal preference, though.

I think there's also usually a space between the doc block and the param list.